omegaconf/basecontainer.py

```diff
@@ -304,7 +304,12 @@ def get_node_value(key: Union[DictKeyType, int]) -> Any:
         assert False
 
     @staticmethod
-    def _map_merge(dest: "BaseContainer", src: "BaseContainer") -> None:
+    def _map_merge(
+        dest: "BaseContainer",
+        src: "BaseContainer",
+        extend_lists: bool = False,
+        allow_duplicates: bool = False,
```

Comment:
I think deduping should not happen by default.
```python
remove_duplicates: bool = False
```
Docs and examples should be updated of course.

Reply:
I'm ok with it and I updated the flag, behavior, docs, tests and examples